Hazel/src/Hazel/Application.cpp

```diff
@@ -78,4 +78,23 @@ namespace Hazel {
 		return true;
 	}
 
-}
\ No newline at end of file
+	std::string Application::CorrectFilePath(const std::string& path)
+	{
+		#if defined(HZ_DEBUG) || defined(HZ_RELEASE)
+			struct stat buffer;
+			int status;
+
+			std::vector<std::string>prepath_string_vector = {"", "./Sandbox/", "../../../Sandbox/"};
+			for (auto path_iter : prepath_string_vector)
+			{
+				std::string modified_path = path_iter + path;
+				status = stat(modified_path.c_str(), &buffer);
```

Comment:
`stat()` is linux specfic function.

Reply:
It's a Posix C function. @LovelySanta tested it for me.

std::filesystem is giving me segfaults on my Linux system. I need to escalate that issue. GNU usually is very careful with testing though. I'll track it down.